### DISCUSSION TITLE
Warning

### DISCUSSION THREAD

**Discussion by Nc2meu2** [RainbowMiner/RainbowMiner#2007] [Q&A]
Why do I keep getting "stat file (NBminer-GPU#01_kawpow_HashRate) was not updated because the value 10.70 is outside fault tolerance 4.42 Gh to 14.73 GH

Any help to fix this will be appreciated

**Comment by RainbowMiner** [comment]
Could you please create a Debug file (open http://localhost:4000 with your browser and click "Debug file" on the left hand side) and upload it here? I will check the situation.

**Reply by Nc2meu2** [reply]
Thank you for checking this out.

**Comment by Nc2meu2** [comment]
[debug_2022-03-16.zip](https://github.com/RainbowMiner/RainbowMiner/files/8271869/debug_2022-03-16.zip)

**Comment by RainbowMiner** [comment]
Weird - the NBminer benchmark seems to have gone wrong. Please delete the file 
`.\Stats\Miners\NVIDIA-NBminer-GPU#01_KawPOW_HashRate.txt`
and check if the warning goes away, after that.

**Reply by Nc2meu2** [reply]
The file was deleted. RainbowMiner was stopped and closed. I opened it and the warning came back, and the file too. When RainbowMiner was checking all the configurations, it displayed a warning and then started mining
![Rainbowminer](https://user-images.githubusercontent.com/101607809/158708644-012132ec-2cdb-473a-b323-a18dc75979b0.png)

**Reply by Nc2meu2** [reply]
Sorry Im not letting you respond but I'm getting this now...
![Rainbowminer new](https://user-images.githubusercontent.com/101607809/158709139-ef052117-54e3-45ca-93cf-27c8f47e98e1.png)

**Reply by RainbowMiner** [reply]
Could you please upload that specific file here?
`.\Stats\Miners\NVIDIA-NBminer-GPU#01_KawPOW_HashRate.txt`

**Comment by RainbowMiner** [comment]
Ahh! NOW I see it. It's a problem with the Fastlane-Benchmark. There has been a very much too high number in the database for that GTX1660 + KawPow + NBminer. I have implemented a preliminary plausibility check, now. Please delete that file again (and maybe update to Powershell Core 7.2.2, too). Now you should get a better number.

Alternatively, disable the Fastlane benchmark now (set `"EnableFastlaneBenchmark": "0",` in config.txt) and then delete that file again. Your rig will then benchmark your GPU locally.

EDIT:
I have just updated the core code with some optimizations regarding the fastlane values. If a value is too far off of the current miner's hashrate, the local stat file will be reset to the current hashrate.

**Reply by Nc2meu2** [reply]
Sorry for the late response. I followed yours instructions and it worked up until the next day. I updated powershell, Raimbowminer got an update as well, and I tried changing some of the settings...i was still getting that warning. I also realized there were too many coins to mine at once, so once the pay out kicked in I reset everything because I'm new and didn't research how to remove a coin from mining.  

I'll update you if I get any warning again. 

Thank you for this awesome program and for being available to answer questions. 

I'll share this program to ppl that may ask about mining.
